GCP/legos/gcp_save_csv_to_google_sheets_v1/gcp_save_csv_to_google_sheets_v1.py

```diff
@@ -0,0 +1,64 @@
+
```

Comment:
Missing copyright

Reply:
The export tool rearranged the file.

Copyright is on line 27